ApplicationLibCode/Application/RiaSummaryCurveDefinition.cpp

```diff
@@ -42,6 +46,8 @@ RiaSummaryCurveDefinition::RiaSummaryCurveDefinition( RimSummaryCase*
                                                       bool                            isEnsembleCurve )
     : m_summaryCase( summaryCase )
     , m_summaryAddress( summaryAddress )
+    , m_summaryCaseX( nullptr )
+    , m_summaryAddressX( RifEclipseSummaryAddress::timeAddress() )
```

Comment:
We should maybe rename m_summaryCase to m_summaryCaseY at some point. But probably a good idea to wait til the feature is complete?

Reply:
Fixed